libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

```diff
@@ -577,3 +581,8 @@ function getSwaggerTypeFromVariableType(variableType: string): string | undefine
       return undefined;
   }
 }
+
```

Comment:
We should check here for aliasing supported know this parameter object, I think should present if it has alias then its open api format